### PR TITLE
Feat contact

### PR DESCRIPTION
**Pull request by austinmargarone**
@brandonetter The only thing that I'm stuck on here is getting the radius on the rectangles to the right of the button on the mobile contact card.

### CODE REVIEW

**Review by brandonetter — DISMISSED** [review]
@austinmargarone  Not sure what you mean by radius of the rectangles to the right of the button on the mobile contact card?
Yours:
image

Design:
image

I'm not sure what you mean?

**Comment by brandonetter** [comment]
@austinmargarone OH I see!

```js
<div className="absolute bottom-[.54rem] right-[3.25rem] shrink-0 opacity-[.02rem]">
            <Image
              className="rounded-br-3xl"
              src="/Rectangle1.svg"
              width={50}
              height={154}
              alt="Rectangle"
            />
          </div>
          <div className="absolute bottom-[.54rem] right-[3.25rem] shrink-0 opacity-[.02rem]">
            <Image
              className="rounded-br-3xl"
              src="/Rectangle2.svg"
              width={50}
              height={154}
              alt="Rectangle"
            />
          </div>
```

You can use `rounded` on specific corners with `rounded-br-` and the like.
See reference here:
https://tailwindcss.com/docs/border-radius

File: components/homepage/Contact.tsx [hunk 1]
```diff
@@ -1,9 +1,135 @@
...
+          </div>
+          <div className="absolute bottom-[.54rem] right-[3.25rem] shrink-0 opacity-[.02rem]">
+            <Image
+              src="/Rectangle2.svg"
+              width={50}
+              height={154}
+              alt="Rectangle"
+            />
+          </div>
+        </div>
```
Comment (brandonetter) [inline components/homepage/Contact.tsx]: Aside from the tips I gave you about border radius - This is little redundant.

One image with properly changed opacity should do fine:

```js
    <div className="absolute bottom-[.54rem] right-[3.25rem] shrink-0 opacity-[0.88]">
            <Image
              className="rounded-br-3xl"
              src="/Rectangle2.svg"
              width={50}
              height={154}
              alt="Rectangle"
            />
          </div>
```

Note here that I changed your opacity from "rem" to just a floating point number.

**Comment by austinmargarone** [comment]
@brandonetter It's not applying any effect on the image.

**Review by brandonetter — CHANGES_REQUESTED** [review]

**Comment by brandonetter** [comment]
@austinmargarone the issue with the code you just tried was that you put the border radius on the div not the image like I did

**Comment by austinmargarone** [comment]
@brandonetter Good to know! That would have been way easier.

**Comment by brandonetter** [comment]
Please integrate my suggestion of not using duplicate images overlaying each other to accomplish this. Use one image and just adjust the opacity. @austinmargarone

**Comment by austinmargarone** [comment]
@brandonetter Implemented. I'm having two issues though:
Mobile the bottom right does not look like the design 
Desktop looks to be skewed down a bit from the top

**Comment by austinmargarone** [comment]
I'm merging this so I can fix all the conflicts and get a PR ready for my next branch. Please still take a look at the following when you get a chance from what you asked me to do with the group photos.

two issue:
Mobile the bottom right does not look like the design
Desktop looks to be skewed down a bit from the top

@brandonetter

**Pull request merged**